src/hooks/useSelectButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+const useSelectButton = () => {
+  const [selectBtn1, setSelectBtn1] = useState<string>('ALL');
+  const [selectBtn2, setSelectBtn2] = useState<string>('받고 싶어한');
```

Comment:
```suggestion
  const [selectBtn1, setSelectBtn1] = useState('ALL');
  const [selectBtn2, setSelectBtn2] = useState('받고 싶어한');
```

타입이 추론될 수 있을 때엔 명시적으로 남기지 않는 게 더 좋아요!